corehq/apps/export/views/new.py

```diff
@@ -159,12 +156,13 @@ def page_context(self):
 
     @property
     def _possible_geo_properties(self):
-        if self.export_type == FORM_EXPORT:
-            return []
-
         if self._is_bulk_export:
             return []
 
+        if self.export_type == FORM_EXPORT:
+            # 'location' is the geo-point in the form metadata
+            return ['location']
```

Comment:
nit: confirming that this does not break for forms that don't have this metadata set?